deployer/README.md

```diff
@@ -42,6 +42,8 @@ This section descripts all the deployment related subcommands the `deployer` can
 │ exec-homes-shell                    Pop an interactive shell with the home directories of the given hub mounted on   │
 │                                     /home                                                                            │
 │ exec-hub-shell                      Pop an interactive shell in the hub pod                                          │
+│ generate-cluster                    Automatically generate the files required to setup a new cluster on a specific   │
```

Comment:
I usually set `COLUMNS=120` before exporting the help file so it all aligns nicely